samples/hello/scripts/generate-keys.py

```diff
@@ -52,12 +51,29 @@ async def main(config_path, api_key, device_name):
     # Create and self-sign a corresponding public key / certificate
     run(f'openssl req -x509 -new -nodes -key "{root_ca_key}" -sha256 -subj "/C=US/CN=Root {ROOT_CA_NAME}" -days 1024 -out "{root_ca_crt}"')
 
+    if upload:
+        # Add (upload) generated CA certificate to Golioth
+        with root_ca_crt.open('rb') as fp:
+            await project.certificates.add(fp.read(), cert_type='root')
+
+@click.command()
+@click.option('--device-id', help="Primary Device Hardware ID (e.g. a serial number)",
```

Comment:
"device-id" and "hardware-id" are probably two separate things in Golioth nomenclature. We should use one name for one thing. E.g. when you list devices with `goliothctl device list` then there is `id:"xxxxxxxxxxxxxxxx"` field shown and then `hardware_ids:"xxxxx"`.